kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,106 @@ This sample demonstrates how to install the Traefik ingress controller to provid
+Now you can send requests to different WebLogic domains with the unique Traefik entry point of different paths, as defined in the route section of the `path-routing.yaml`.
 ```
-$ curl http://${HOSTNAME}:30305/domain1/
-$ curl http://${HOSTNAME}:30305/domain2/
+# Get the ingress controller web port
+$ export LB_PORT=$(kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.ports[?(@.name=="web")].nodePort}')
+$ curl http://${HOSTNAME}:${LB_PORT}/domain1/
+$ curl http://${HOSTNAME}:${LB_PORT}/domain2/
 ```
-#### Install a TLS-enabled Ingress
-This sample demonstrates accessing the two WLS domains using an HTTPS endpoint and the WLS domains are protected by different TLS certificates.
-
-To make this sample work, you need to enable the TLS endpoint in the Traefik operator. If you use the `values.yaml` file in the same folder as this README, the TLS endpoint is already enabled.
+#### Host-based secured routing 
+This sample demonstrates how to access an application on two WebLogic domains using an HTTPS endpoint. Install a TLS-enabled Traefik [IngressRoute](https://docs.traefik.io/routing/providers/kubernetes-crd/#kind-ingressroute).
 
-First, you need to create two secrets with TLS certificates, one with the common name `domain1.org`, the other with the common name `domain2.org`. We use `openssl` to generate self-signed certificates for demonstration purposes. Note that the TLS secret needs to be in the same namespace as the WLS domain.
+First, you need to create two secrets with TLS certificates, one with the common name `domain1.org`, the other with the common name `domain2.org`. We use `openssl` to generate self-signed certificates for demonstration purposes. Note that the TLS secret needs to be in the same namespace as the WebLogic domain.
 ```
 # create a TLS secret for domain1
 $ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/tls1.key -out /tmp/tls1.crt -subj "/CN=domain1.org"
-$ kubectl create secret tls domain1-tls-cert --key /tmp/tls1.key --cert /tmp/tls1.crt
-
+$ kubectl -n weblogic-domain1 create secret tls domain1-tls-cert --key /tmp/tls1.key --cert /tmp/tls1.crt
 # create a TLS secret for domain2
 $ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/tls2.key -out /tmp/tls2.crt -subj "/CN=domain2.org"
-$ kubectl -n test1 create secret tls domain2-tls-cert --key /tmp/tls2.key --cert /tmp/tls2.crt
-```
-Then deploy the TLS Ingress.
-```
+$ kubectl -n weblogic-domain2 create secret tls domain2-tls-cert --key /tmp/tls2.key --cert /tmp/tls2.crt
+
+# Deploy a TLS IngressRoute.
 $ kubectl create -f samples/tls.yaml
+ingressroute.traefik.containo.us/traefik-tls-1 created
+ingressroute.traefik.containo.us/traefik-tls-2 created
 ```
-Now you can access the two WLS domains with different hostnames using the HTTPS endpoint.
+Now you can access the application on the WebLogic domain with the hostname in HTTP header.
```

Comment:
hostname in HTTP header. -> host name in the HTTP header.

Reply:
done